### ISSUE TITLE
Add pool names to stats in logs

### ISSUE THREAD

**Issue by civilizatorr** [Sniffdog/Sniffdogminer#28]
Thanks.

**Comment by Sniffdog** [comment]
All timestamp profit is by each pool... what exactly are you looking for?

**Comment by civilizatorr** [comment]
![sniff](https://user-images.githubusercontent.com/34585669/34067425-99e9633a-e236-11e7-8eb2-22f8163d0431.PNG)
You see, some algos are mtnioned twice - because they were mined on different pools. Please add names of pools to this list.

**Comment by Sniffdog** [comment]
yeah ummmm underneath "Command" it should read the the bat to each miner and location.... this is all very cool but I think it's not Sniffdog.....

Also, you have two miners open in the "Miners" folder for blake2s...

Maybe try to download latest version of Sniffdog from the release notes

**Comment by civilizatorr** [comment]
Yes, I was using modified ahashpool version. Thanks.
By the way, your EWBF with default settings gets only 500sols from 1080Ti. Has to change intensity to 63, and now its good 800sols from the card.

**Comment by Sniffdog** [comment]
Thanks for the note... I'm currently waiting for the new ewbf as well
